models/source_list.py

```diff
@@ -444,7 +430,7 @@ def calc_aper_cor( self, aper_num=0, inf_aper_num=None, min_stars=20 ):
 
         bigflux, bigfluxerr = self.apfluxadu( apnum=inf_aper_num )
         smallflux, smallfluxerr = self.apfluxadu( apnum=aper_num )
-        wgood = self.is_star & self.good & ( bigflux > 5.*bigfluxerr ) & ( smallflux > 5.*smallfluxerr )
+        wgood = self.good & ( bigflux > 5.*bigfluxerr ) & ( smallflux > 5.*smallfluxerr )
```

Comment:
Need to leave is_star in here -- we only want to be doing aperture corrections on PSFs.  It's better to not use all the stars (because some got omitted) than include things that aren't stars.  In the former case, it will (ideally) just increase the uncertainty.  In the latter case, it will systematically bias the aperture correction.

Reply:
I am leaving this out and will fix it when addressing issue #154